internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/controller/TestHybridStoreRepartitioningWithMultiDataCenter.java

```diff
@@ -159,12 +191,23 @@ public void testHybridStoreRepartitioning() {
         .setHybridOffsetLagThreshold(1000);
     TestWriteUtils.updateStore(storeName, parentControllerClient, updateStoreParams);
 
+    for (ControllerClient controllerClient: childControllerClients) {
+      TestUtils.waitForNonDeterministicAssertion(30, TimeUnit.SECONDS, () -> {
+        StoreInfo storeInfo = controllerClient.getStore(storeName).getStore();
+        Assert.assertEquals(storeInfo.getLargestUsedRTVersionNumber(), 1);
+      });
+    }
+
     // create new version by doing an empty push
     parentControllerClient
         .sendEmptyPushAndWait(storeName, Utils.getUniqueString("empty-push"), 1L, 60L * Time.MS_PER_SECOND);
 
     for (ControllerClient controllerClient: childControllerClients) {
-      Assert.assertEquals(controllerClient.getStore(storeName).getStore().getCurrentVersion(), 1);
+      StoreInfo storeInfo = controllerClient.getStore(storeName).getStore();
+      Assert.assertEquals(
+          storeInfo.getHybridStoreConfig().getRealTimeTopicName(),
+          storeName + "_v1" + Version.REAL_TIME_TOPIC_SUFFIX);
+      Assert.assertEquals(storeInfo.getCurrentVersion(), 1);
```

Comment:
Let's also add version level RT name verification

Reply:
it is already there as the last assert in this block